### DISCUSSION TITLE
if the backup files from other velero , can  velero restore  from local file or file imported manually  to  the same destination backup  bucket

### DISCUSSION THREAD

**Discussion by wxcoin** [vmware-tanzu/velero#7206] [Community support Q&A] [answered]
[root@k8s-master2]# velero restore create --from-backup laboratory-ns-backup --wait
An error occurred: backups.velero.io "laboratory-ns-backup" not found
is there any config to edit then i can restore from the file imported to the same backup bucket that  backup from other velero ? 
or  from local backup file

i uploaded the backup file manually to the  velero backup bucket and chage the file name ,the velero doesn't recognize
[root@k8s-master2 ]# velero backup get|grep lab

**Comment by sseago** [comment, marked as answer]
If you add the bucket that contains the backup to the current velero instance, it should import it. 

Your manual update probably didn't work because you didn't upload everything that Velero created for the backup, or you didn't wait for the backup sync controller to run its next sync. However, in general, I'd recommend against copying backup files around, as there's a good chance something will be missed.

But velero explicitly supports backing up in one cluster and restoring in another -- all you need to do is make sure that a backup storage location is defined in both clusters pointing to the same bucket.
